registers/trillian_client/client.go

```diff
@@ -0,0 +1,81 @@
+package trillian_client
+
+import (
+	"context"
+	"log"
+
+	"github.com/google/trillian"
+	"google.golang.org/grpc"
+)
+
+const CHUNK = 10
+
+type LogScanner interface {
```

Comment:
Need doc comments on exported types, and their exported methods.

Reply:
Done